.github/workflows/dapr-perf.yml

```diff
@@ -436,90 +436,9 @@ jobs:
       - name: Deploy test components
         if: env.TEST_PREFIX != ''
         run: make setup-test-components
-      - name: Run Perf test service_invocation_http
+      - name: Run all performance tests
         if: env.TEST_PREFIX != ''
-        env:
-          DAPR_PERF_QPS: 1000
```

Comment:
Is there a way to still have the ability to manually set these env vars to override the values that are now hardcoded?

Use case is if one is testing perf tests locally and wants to set different targets for whatever reason (e.g. because the hardware is different)

Reply:
Yes, I mentioned that in the related issue, the only difference is that the hardcoded values will be used but the environment variables yet have precendence

Reply:
Oh alright. I missed that, thanks for clarifying